docs/content/concepts/transfers.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: Transfers
+---
+
+Everything on Sui is an object and your smart contracts are inevitably going to need to move those objects around the network, transferring them from one owner to another. The topics in this section explore the options you have on Sui around transferring objects on the network.
+
+## Custom Transfer Rules
+
+On Sui, you can create custom transfer rules for objects that define the transfer conditions that must be met for a valid transfer operation. The object you want to create custom transfer rules for cannot have the `store` ability, as that ability provides unrestricted access to that object without having to go through the module that defines it. 
```

Comment:
nit: overuse of the word "transfer"? Also clarifying what "unrestricted access" means in this case (my amended copy probably needs to have links introduced for references to `public_transfer` and `TransferObjects`).

```suggestion
On Sui, you can create custom transfer rules for objects that define the conditions that must be met for a valid transfer operation. The object you want to create custom transfer rules for cannot have the `store` ability, as that ability enables unrestricted transfers for the type via `sui::transfer::public_transfer` or the `TransferObjects` transaction command. 
```